src/ostree/ot-main.c

```diff
@@ -48,6 +50,28 @@ static GOptionEntry repo_entry[] = {
   { NULL }
 };
 
+static gboolean
+parse_lock_timeout (const char  *option_name,
+                    const char  *value,
+                    gpointer     data,
+                    GError     **error)
+{
+  char *endptr = NULL;
+  gint64 lock_timeout = g_ascii_strtoll (value, &endptr, 0);
+  if ((lock_timeout < -1) ||
+      (lock_timeout > G_MAXINT) ||
+      (lock_timeout == 0 && endptr == value))
```

Comment:
As with the other `g_ascii_strtoll()` call, this should check for `*endptr != '\0'` as another error condition.

Reply:
Will fix after #1343.